Reorganize 3

ljanicke opened the pull request.

ljanicke committed:
Reorganize landscape.yml

ljanicke committed:
Update settings.yml

ljanicke committed:
Merge pull request #1 from ljanicke/ljanicke-reorganize

Reorganize landscape.yml

ljanicke committed:
Merge pull request #2 from ljanicke/ljanicke-reorganize-1

Update settings.yml

ljanicke committed:
Removed extra entries

ljanicke committed:
Resize categories in settings.yml

ljanicke committed:
Update settings.yml

ljanicke committed:
Removed empty IPV subcategory

ljanicke committed:
Resize categories in settings.yml 2

ljanicke committed:
Merge branch 'master' into resolve-merge-conflict

ljanicke committed:
Merge pull request #3 from ljanicke/resolve-merge-conflict

Resolve merge conflict

ljanicke committed:
Testing cols argument

ljanicke committed:
Testing rows argument

ljanicke committed:
Add NREL tag

ljanicke committed:
Testing NREL tag

ljanicke committed:
Testing existence of card size code

ljanicke committed:
Moved NREL tag

ljanicke committed:
Testing NREL tag again

ljanicke committed:
Change NREL tag color, add NREL page

ljanicke committed:
Add all NREL tags, create NREL landscape

ljanicke committed:
Fix subcategory typo

ljanicke committed:
Remove duplicate in landscape.yml

ljanicke committed:
Update settings.yml

ljanicke committed:
Update category name

ljanicke committed:
Remove info in NREL cat

Seeing if the extra info is the issue with the previews being created

ljanicke committed:
Add argument to NREL cat

ljanicke committed:
Add SVG files for new NREL items

ljanicke committed:
Add NREL entries to landscape and NREL cat

ljanicke committed:
Remove duplicates again

ljanicke committed:
Add more subcat to NREL cat

ljanicke committed:
Replace SVG files for new NREL entries

ljanicke committed:
Change color of categories

ljanicke committed:
Resize categories

ljanicke committed:
Change height

ljanicke committed:
Resize categories 2

ljanicke committed:
Shift categories vertically

ljanicke committed:
Edit GH Actions

ljanicke committed:
Add NREL logos with names

ljanicke committed:
Resize NREL cat

ljanicke committed:
Remove overlap of NREL cat

ljanicke committed:
Delete typo in settings.yml

ljanicke committed:
Resize NREL cat

ljanicke committed:
Remove edit from GH actions

ljanicke committed:
Add data repo cat

ljanicke committed:
Remove overlap of main cats

ljanicke committed:
Change data cat to datasets

ljanicke committed:
Add data repo cat

ljanicke committed:
Correct spacing in landscape.yml

ljanicke committed:
Edit main cats

ljanicke committed:
Edit main cats

ljanicke committed:
Change NREL color

ljanicke committed:
Add new entries to landscape.yml

ljanicke committed:
Add photos of new entries

ljanicke committed:
Fix typo in landscape.yml

ljanicke committed:
Change data owner in new entries

ljanicke committed:
Remove ' from new descriptions

ljanicke committed:
Remove new entries

ljanicke committed:
Add AFDC

ljanicke committed:
Change data_format for AFDC

ljanicke committed:
Add NREL and livewire entries

ljanicke committed:
Define mobility better

ljanicke committed:
Define mobility better

ljanicke committed:
Edit NREL cat

ljanicke committed:
Edit CNCF relations filter

ljanicke committed:
Add some relations back in

ljanicke committed:
Add more relations back

ljanicke committed:
Revert to CNCF relations

ljanicke committed:
Fix indentation

ljanicke committed:
Remove some memberships

ljanicke committed:
Change label of hosted values

ljanicke committed:
Remove memberships

ljanicke committed:
Add children to memberships

ljanicke committed:
Change CNCF relation to NREL relation

ljanicke committed:
Remove ads

ljanicke committed:
Change ad

ljanicke committed:
Remove twitter from settings.yml

ljanicke committed:
Add twitter back

ljanicke committed:
Alphabetize subcategories

ljanicke committed:
Change data to resource & remove mobility definition

in settings.yml

shankari commented:
Going to keep this unchanged while we ask people for the feedback side by side.
Also, the check is actually failing?

ljanicke committed:
Change more "data" to "resources" in settings

ljanicke commented:
Sounds good. The check was not actually failing.

ljanicke committed:
Resize cats

ljanicke committed:
Resize NREL cats 2

ljanicke committed:
Resize Sim cat

ljanicke committed:
Move SigAlert

ljanicke committed:
Change cat names

ljanicke committed:
Change cat names

ljanicke committed:
Change cat names

ljanicke committed:
Change cat names

ahcyip committed:
Update settings.yml

misc updates

ljanicke committed:
Merge pull request #4 from ahcyip/patch-2

Update settings.yml

ljanicke committed:
Testing three levels of subcategories

ljanicke committed:
Reverting changes

ljanicke committed:
Update settings.yml

ljanicke committed:
Reorganize Tools cat

ljanicke committed:
Split Sim cat

ljanicke committed:
Add new entries without logos

ljanicke committed:
Add placeholder image

ljanicke committed:
Add placeholder.svg to entries

ljanicke committed:
Move cats

ljanicke committed:
Remove mini NREL landscape

extra:
    tab_index: 1
    url: NREL
    name: NREL
    short_name: NREL
    title: NREL Projects Landscape
    category: NREL Projects
    fullscreen_header: Missing data or request a change?  Submit a pull request!
    elements:
      - type: HorizontalCategory
        category: Data Analytics
        rows: 1
        width: 850
        height: 100
        top: 0
        left: 0
        color: rgb(212,82,40)
      - type: HorizontalCategory
        category: Infrastructure
        rows: 1
        width: 850
        height: 100
        top: 125
        left: 0
        color: rgb(84,169,68)
      - type: HorizontalCategory
        category: Vehicles and Mobility
        rows: 1
        width: 850
        height: 100
        top: 250
        left: 0
        color: rgb(239,170,33)

ljanicke committed:
Remove entries of mini NREL landscape

 - category:
    name: NREL Projects
    subcategories: 
      - subcategory:
        name: Data Analytics
        items:
          - item:
            name: Mobility Energy Productivity Metric (MEP)
            homepage_url: 'https://www.nrel.gov/transportation/mobility-energy-productivity-metric.html'
            logo: mep.svg
          - item:
            name: Transportation Secure Data Center (TSDC)
            homepage_url: 'https://www.nrel.gov/transportation/secure-transportation-data/index.html'
            logo: tsdc.svg
      - subcategory:
        name: Infrastructure
        items:
          - item:
            name: Electric Vehicle Infrastructure - Energy Estimation and Site Optimization Tool (EVI-EnSite)
            homepage_url: 'https://www.nrel.gov/transportation/evi-ensite.html'
            logo: evi-ensite.svg
          - item:
            name: Electric Vehicle Infrastructure - Financial Analysis Scenario Tool
            homepage_url: 'https://www.nrel.gov/transportation/evi-fast.html'
            logo: evi-fast.svg
          - item:
            name: Electric Vehicle Infrastructure Projection Tool Lite
            homepage_url: 'https://afdc.energy.gov/evi-pro-lite'
            logo: evipro-lite.svg
          - item:
            name: Hydrogen Financial Analysis Scenario Tool
            homepage_url: 'https://www.nrel.gov/hydrogen/h2fast.html'
            logo: h2fast.svg
          - item:
            name: Hydrogen Filling Simulation Tool
            homepage_url: 'https://www.nrel.gov/hydrogen/h2fills.html'
            logo: h2fills.svg
          - item:
            name: Scenario Evaluation and Regionalization Analysis Model (SERA)
            homepage_url: 'https://www.nrel.gov/hydrogen/sera-model.html'
            logo: sera.svg
          - item:
            name: Wireless Power Transfer Simulator
            homepage_url: 'https://www.nrel.gov/transportation/data-tools.html'
            logo: wtpsim.svg
      - subcategory: 
        name: Vehicles and Mobility
        items:
          - item: 
            name: Automotive Deployment Options Projection Tool (ADOPT)
            homepage_url: 'https://www.nrel.gov/transportation/adopt.html'
            logo: adopt.svg
          - item:
            name: Highly Integrated Vehicle Ecosystem Framework (HIVE)
            homepage_url: 'https://www.nrel.gov/transportation/hive.html'
            logo: hive.svg
          - item: 
            name: RouteE!
            homepage_url: 'https://www.nrel.gov/transportation/route-energy-prediction-model.html'
            logo: routee.svg
          - item:
            name: Transportation Energy & Mobility Pathway Options (TEMPO)
            homepage_url: 'https://www.nrel.gov/transportation/tempo-model.html'
            logo: tempo.svg

ljanicke committed:
Add more logos

ljanicke committed:
Move logo 1

ljanicke committed:
Move logo 2

ljanicke committed:
Move logo 3

ljanicke committed:
Move logo 4

ljanicke committed:
Move logo 5

ljanicke committed:
Add logos to some placeholders

ljanicke committed:
Add more logos

ljanicke committed:
Add logos to placeholders

ljanicke committed:
Resize cats

ljanicke committed:
Fix entries to improve filter consistency

ljanicke committed:
Edit entries missed for filter improvements

ljanicke commented:
Should be ready to go!
